### DISCUSSION TITLE
Why github version ≠ chrome or Firefox version ?

### DISCUSSION THREAD

**Discussion by Miala-python** [ScratchAddons/ScratchAddons#3363] [Help] [answered]
I have see that the new version is available on GitHub but not on the Chrome Web store... Why?

_Sorry, I don't speak English very well: I'm French._

**Comment by CST1229** [comment, marked as answer]
Both the Chrome and Firefox store versions need to go through review by Google/Mozilla to be available on their respective stores for each update, to make sure the extension is safe. However, GitHub releases are available immediately since they don't need to get reviewed at all.

**Reply by Miala-python** [reply]
Thanks 👍😊 !

**Reply by Samq64** [reply]
I thought Scratch Addons didn't need to go through review by Mozilla before. I'm pretty sure it used to be available right away.

**Reply by WorldLanguages** [reply]
Yeah we've recently started to release both at the same time. Mozilla has no option to let us release when we want, it's only automatically after review (review often takes 10 minutes or less) so instead we just upload Mozilla when Chrome approves.

**Reply by TheMapotakes** [reply]
Yes

**Comment by Samq64** [comment]
Does it usually take over 2 days for chrome to get approved?

**Reply by cobaltt7** [reply]
there was a bug in 1.19.0 so we canceled the release. we are releasing 1.19.1 rn.

**Comment by TheMapotakes** [comment]
Ok
